integration-tests/pkg/mock_sensor/server.go

```diff
@@ -47,7 +46,7 @@ type MockSensor struct {
 	processLineages map[string]LineageMap
 	processMutex    sync.Mutex
 
-	connections  map[string]ConnMap
+	connections  map[string][]types.NetworkInfo
```

Comment:
it might be worth retaining the type alias for consistency with the other maps (but renamed to something else like `ConnList`)

Reply:
I think other PRs should change the other maps to lists as well. I think naming a list `ConnList` will just obfuscate things.

Reply:
I think other PRs should change the other maps to lists as well. I think naming a list `ConnList` will just obfuscate things.